homeassistant/components/binary_sensor/mercedesme.py

```diff
@@ -0,0 +1,107 @@
+"""
+Support for Mercedes cars with Mercedes ME.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/mercedesme/
+"""
+import logging
+import datetime
+
+from homeassistant.components.binary_sensor import (BinarySensorDevice)
+from homeassistant.components.mercedesme import (
+    DATA_MME, MercedesMeEntity, BINARY_SENSORS)
+
+DEPENDENCIES = ['mercedesme']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the sensor platform."""
+    data = hass.data[DATA_MME].data
+
+    if not data.cars:
+        _LOGGER.error("setup_platform data.cars is none")
+        return
+
+    devices = []
+    for car in data.cars:
+        for dev in BINARY_SENSORS:
+            devices.append(MercedesMEBinarySensor(
+                data, dev, dev, car["vin"], None))
+
+    add_devices(devices, True)
+
+
+class MercedesMEBinarySensor(MercedesMeEntity, BinarySensorDevice):
+    """Representation of a Sensor."""
+
+    @property
+    def is_on(self):
+        """Return the state of the binary sensor."""
+        retval = True if bool(self._state == "On") else False
+        return retval
+
+    @property
+    def state(self):

```

Comment:
`BinarySensorDevice` already defines `state`. We can get rid of this whole function definition.